middleware/cors/cors.go

```diff
@@ -81,12 +81,6 @@ func New(config ...Config) fiber.Handler {
 		cfg = config[0]
 
 		// Set default values
-		if cfg.Next == nil {
-			cfg.Next = ConfigDefault.Next
-		}
-		if cfg.AllowOrigins == "" {
```

Comment:
why you remove that ?
default allowOrigins is "*" and should be used, when nothing is added in the custom user config

Reply:
Yeah you are right. I made a mistake :smile: